File: src/AC/2678/2678.cpp
#include <string>
#include <vector>

class Solution {
public:
    int countSeniors(std::vector<std::string>& details) {
        int count = 0;
        for (auto detail : details) {
            int age = std::stoi(detail.substr(11, 2));
            if (age > 60) {
                count++;
            }
        }

        return count;
    }
};

File: src/AC/2678/2678.rs
impl Solution {
    pub fn count_seniors(details: Vec<String>) -> i32 {
        let mut cnt = 0;
        for i in 0..details.len() {
            let age: i32 = details[i][11..13].parse().unwrap();
            if age > 60 {
                cnt += 1;
            }
        }

        cnt
    }
}
